vmtest/vm.py

```diff
@@ -96,6 +97,57 @@
 "$BUSYBOX" hostname "$HOSTNAME"
 "$BUSYBOX" ip link set lo up
 
+run_command() {{
+    set +e
+    "$BUSYBOX" sh -c {command}
+    rc=$?
+    set -e
+    return $rc
+}}
+
+if "$BUSYBOX" [ -n "${{crashkernel:-}}" ]; then
+    # This isn't great: it's specific to x86-64, both by virtue of the syscall
+    # number and because kexec_file_load isn't implemented on many
+    # architectures, especially on older kernels. Also, embedding the script as
+    # a string here is ugly, but it avoids the need to locate yet another file.
+    "$PYTHON" -c '
+import ctypes
+import os
+import re
+import sys
+
+SYS_kexec_file_load = 320  # On x86-64.
+KEXEC_FILE_ON_CRASH = 2
+KEXEC_FILE_NO_INITRAMFS = 4
+
+syscall = ctypes.CDLL(None, use_errno=True).syscall
+syscall.restype = ctypes.c_long
+
+with open("/proc/cmdline", "rb") as f:
+    cmdline = f.read().rstrip(b"\n")
+    cmdline = re.sub(rb"(^|\s)crashkernel=\S+", b"", cmdline)
+    cmdline += b" nosmp nokaslr"
```

Comment:
We should probably comment on why these are necessary.

Reply:
Addressed in #140